ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,57 @@
-﻿namespace ConsoleApp1
-{
-	internal class Program
-	{
-		static void Main(string[] args)
-		{
-		}
-	}
+﻿// metod returns max value amoung all parameters
+var t = MaxValueReturn(1, 3);
+Console.WriteLine("Max value is: "+ t);
+// metod returns min value amoung all parameters
+var k = MinValueReturn(4, 2);
+Console.WriteLine("Min value is: " + k);
+//TrySumIfOdd
+int x;
+int y;
+int z;
+var c = TrySumIfOdd(out x, out y, out z);
+Console.WriteLine($"result: {c}, sum: {z}");
+//The last task
+int resultMaxValue = MaxValue(5, 9, 6);
+Console.WriteLine("MaxValue:" + resultMaxValue);
+int resultMinValue = MinValue(4, 2, 1);
+Console.WriteLine("MinValue:"+ resultMinValue);
+
+static int MaxValueReturn(int a, int b)
+{
+    if(a > b)
```

Comment:
you can use either ternary operator or Math.Max(a,b)

Reply:
done